folly/concurrency/test/AtomicSharedPtrPerformance.cpp

```diff
@@ -13,6 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+// AtomicSharedPtr-detail.h does not work with libc++, so skip these tests for libc++
+#include <folly/Portability.h>
+#if not defined(FOLLY_USE_LIBCPP)
```

Comment:
`if !defined` or `ifndef`?

Reply:
Switched to `#ifndef` -- sorry about that.